website/docs/components/app-header/index.md

```diff
@@ -32,5 +32,6 @@ status:
 </section>
 
 <section data-tab="Version history">
+  @include "partials/version-history/4.10.0.md"
   @include "partials/version-history/4.8.0.md"
```

Comment:
thought: not sure we want to maintain these component-level version docs in perpetuity 🤔  @didoo any thoughts on this?

Reply:
My though when implementing this feature was that yes, we could maintain them

- it's an easy reference to see how a component evolved, and honestly they don't add much weight to the page itself (and if a user don't click on the tab, they don't see the content anyway)
- the alternative would be the release notes on the website, but 1) they are truncated anyway and 2) it's harder to find content specific for a component